README.md

```diff
@@ -33,8 +66,13 @@ https://k6.io/docs/using-k6/metrics/
 * http_req_sending - Time spent sending data to the remote host
 * http_req_receiving - Time spent receiving response data from the remote host
 
-### Supported countries, transport modes
-Countries:
+### Supported Countries
+
+To see a list of all supported countries, send **map-info** **GET** request ([Postman](https://docs.traveltime.com/api/start/postman-collection#)) to:
+- https://api-dev.traveltimeapp.com for dev
```

Comment:
you can remove dev. It's our public repository

Reply:
removed, left only prod link